Services/UI/OJS.Services.Ui.Models/Contests/ParticipantResultServiceModel.cs

```diff
@@ -0,0 +1,37 @@
+namespace OJS.Services.Ui.Models.Contests;
+
+using AutoMapper;
+using OJS.Data.Models.Participants;
+using SoftUni.AutoMapper.Infrastructure.Models;
+using System;
+using System.Linq;
+
+public class ParticipantResultServiceModel : IMapExplicitly
+{
+    public int ContestId { get; set; }
+
+    public int CompetePoints { get; set; }
+
+    public int PracticePoints { get; set; }
+
+    public DateTime CreatedOn { get; set; }
+
+    public void RegisterMappings(IProfileExpression configuration)
+        => configuration.CreateMap<Participant, ParticipantResultServiceModel>()
+            .ForMember(
+                d => d.PracticePoints,
+                opt =>
+                    opt.MapFrom(src => src.Scores
+                        .Where(s => !s.IsOfficial)
+                        .Sum(s => s.Points)))
+            .ForMember(
+                d => d.CompetePoints,
+                opt =>
+                    opt.MapFrom(src => src.Scores
+                        .Where(s => s.IsOfficial)
+                        .Sum(s => s.Points)))
```

Comment:
I think the scores will always be either official or unoficcial, as different participants are created for compete and practice modes, hence p.Points will be only the points of the official/unofficial participants and we cannot calculate both for a single Participant